### DISCUSSION TITLE
Quick Support portable

### DISCUSSION THREAD

**Discussion by egs2003** [rustdesk/rustdesk#7102] [Q&A] [answered]
If I distribute the exe as a QuickSupport variant, UAC is queried at startup. However, in order not to confuse my inexperienced "customer" even more, it would be nice if the prompt for installation was not displayed underneath. I would like to offer the Quick Support module exclusively as a portable version. Are there any options?

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/8981#discussioncomment-10263847
